internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,6 +1,23 @@
-FROM hashicorp/terraform:light
-ENV TF_IN_AUTOMATION=true
+FROM hashicorp/terraform:light as terraform
+
+FROM python:3-alpine
+
+# required by gcloud SDK
+RUN apk add curl
+
+ENV GCLOUD_SDK_VERSION 367.0.0
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
+RUN curl "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_SDK_VERSION-linux-x86_64.tar.gz" > /tmp/google-cloud-sdk.tar.gz \
```

Comment:
Please remove the `/tmp/google-cloud-sdk.tar.gz` file as it isn't required anymore.